implants/lib/eldritch/src/random/string_impl.rs

```diff
@@ -0,0 +1,49 @@
+use anyhow::Result;
+use rand::{distributions::Alphanumeric, Rng}; // 0.8
+
+pub fn string(length: u64) -> Result<String> {
+    let s: String = rand::thread_rng()
```

Comment:
This seems okay.
I noticed in the bool impl the rng is initialized separately.

let mut rng = rand_chacha::ChaCha20Rng::from_entropy();


Is there a reason to do it one way or another?

Reply:
I'm by no means an expert, but [this documentation](https://docs.rs/rand/latest/rand/rngs/index.html#our-generators) made me think that using the thread_rng is more secure since it re-seeded? I don't have a strong preference either way though and am happy to change if that is the preferred implementation